### DISCUSSION TITLE
Hiding certain page content

### DISCUSSION THREAD

**Discussion by FlipH19Switch** [oleeskild/obsidian-digital-garden#566] [Q&A]
Is it possible to hide certain content within a note so that it does not show up on the corresponding webpage?

Some of my pages have a lot of field info (column ID's) that's used by the DB Folder-Dataview plugin combo. This looks like a mess on the webpage and may show links to other pages that I don't want to be visible.

Here's an example of what it looks like at the bottom of a page:

Field1:: link1, link2, link3
Field2:: link4, link4, link6
Field3:: link7, link8, link9

I'd like to hide everything below and including Field1 if possible. There's a [DB Folder request](https://github.com/RafaelGB/obsidian-db-folder/issues/1002) to move these fields into the frontmatter, but idk if that will ever be done.

**Comment by rokville** [comment]
Using Custom Filters (under Advanced) can accomplish this using regex.
For example, you could use the following filter (with no replacement)
:::hidden(.*\n)*?:::

in your example, place the ":::hidden" before the text to be excluded and ":::" after it, like this:

This text above will be seen
:::hidden
Field1:: link1, link2, link3
Field2:: link4, link4, link6
Field3:: link7, link8, link9
:::
This text will be seen also

Of course, you can make the replacement triggers whatever you like, but because it's regex, you want to be careful.

**Reply by FlipH19Switch** [reply]
Thanks for the suggestion. Is it possible to delete everything everything after ":::hidden" instead of everything between ":::hidden" and ":::"? The DB Folder plugin automatically adds new fields to the bottom of the page, so they will not stay contained within the ":::hidden:::" boundaries unfortunately.

I've tried a few things but they did not work:
:::hidden.*
:::hidden(.\n)?
:::hidden(.\n)?.*

**Reply by bcbrookman** [reply]
I needed to use `:::hidden(.*\n)*:::` instead. I don't think it matters, but I'm running 2.65.0.

if you want to replace everything through the end of the page, `:::hidden(.*\n)*.*` also worked for me. As mentioned, be careful, but using an online regex testing tool first can help.
